integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/IntegrasjonspunktImpl.java

```diff
@@ -125,6 +126,18 @@ public GetCanReceiveMessageResponseType getCanReceiveMessage(@WebParam(name = "G
     @Override
     public PutMessageResponseType putMessage(PutMessageRequestType request) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "putMessage" is 11 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)